rocketpy/Function.py

```diff
@@ -3025,6 +3025,7 @@ def __get__(self, instance, owner=None):
                     )
 
                 val.__doc__ = self.__doc__
+                val.__cached__ = True
```

Comment:
Great solution to mark cached attributes, I would only check if there are not any conflicts with this [standard usage](https://docs.python.org/3/reference/import.html#__cached__).

Reply:
This link that you pointed out is related to python modules.
It is useful to show to the user where the module is located in the user's machine, specially when working with virtual environments.

Here is an example:

```bash
C:\Users\guiga>python
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> numpy.__cached__
'C:\\Users\\guiga\\AppData\\Roaming\\Python\\Python310\\site-packages\\numpy\\__pycache__\\__init__.cpython-310.pyc'
>>> numpy.__file__
'C:\\Users\\guiga\\AppData\\Roaming\\Python\\Python310\\site-packages\\numpy\\__init__.py'
>>>
```

On the other hand, what Giovani has created here is a `__cached__` attribute to the funcifyed methods.
This will allow us to know if the method was cached in the class by the funcify decorator.

The `__cached__` attribute is not being assigned to any module, therefore there are not any conflicts here.

Actually, I think it is pretty intuitive using the name `__cached__` here, since it keeps a degree of consistency with the Python conventions. 

See an example:
![image](https://github.com/RocketPy-Team/RocketPy/assets/63590233/5397fb0e-3c7b-409e-b1a5-2323c14507a5)